src/coreclr/jit/compiler.cpp

```diff
@@ -460,6 +460,584 @@ Compiler::Compiler(ArenaAllocator*       arena,
     info.compUsesAsyncContinuation = false;
 }
 
+#ifdef FEATURE_HW_INTRINSICS
+#if defined(TARGET_XARCH)
+//------------------------------------------------------------------------
+// X64VersionOfIsa: Gets the corresponding 64-bit only InstructionSet for a given InstructionSet
+//
+// Arguments:
+//    isa -- The InstructionSet ID
+//
+// Return Value:
+//    The 64-bit only InstructionSet associated with isa
+static CORINFO_InstructionSet X64VersionOfIsa(CORINFO_InstructionSet isa)
```

Comment:
nit: All of these could have stayed in the respective `hwintrinsicxarch.cpp` and `hwintrinsicarm64.cpp` files

We already implement other `Compiler::*` methods in such files, so we should've been able to keep the diffs minimal by just changing `lookupInstructionSet` to `Compiler::lookupInstructionSet` and similar for `lookupIsa`